pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -230,7 +237,7 @@ stdenv.mkDerivation {
       '')
 
    + stdenv.lib.optionalString withManual ''# Install man pages and Info manual
-       make -j $NIX_BUILD_CORES -l $NIX_BUILD_CORES USE_ASCIIDOCTOR=1 PERL_PATH="${buildPackages.perl}/bin/perl" cmd-list.made install install-html install-info \
+       make -j $NIX_BUILD_CORES -l $NIX_BUILD_CORES USE_ASCIIDOCTOR=1 PERL_PATH="${buildPackages.perl}/bin/perl" cmd-list.made install install-html \
```

Comment:
Will we still have manpages after this change? It looks a bit weird not seeing "normal" man pages listed here unless they are covered by "install".

Reply:
yes, those are part of the `install` target in `-C Documentation`